pkg/infinispan/version/version.go

```diff
@@ -70,6 +70,13 @@ func (o Operand) LT(other Operand) bool {
 	return o.UpstreamVersion.LT(*other.UpstreamVersion)
 }
 
+// Hardcoding first version that bans cacheservice
+var minVerBansCacheService = semver.MustParse("15.0.0")
+
+func (o Operand) BansCacheService() bool {
```

Comment:
There's no need for us to check the Upstream Operand version as we're removing the cache service functionality from the Operator completely, so it won't work with any of the Operand versions.